src/LiquidityPool.sol

```diff
@@ -90,6 +90,10 @@ contract LiquidityPool is Auth, IERC7540 {
         emit File(what, data);
     }
 
+    function recoverTokens(address to, address token, uint256 amount) external auth {
```

Comment:
Would be cleaner if the order of the params of the method matches the order of the transfer func
```suggestion
    function recoverTokens(address token, address to, uint256 amount) external auth {
```